data/mapping/mapping.json

```diff
@@ -72,13 +65,12 @@
           }
         },
         "doctors": {
-          "type": "text",
-          "analyzer": "gp_name_analyzer",
-          "fields": {
-            "keyword": {
-              "type": "keyword",
-              "ignore_above": 256
-            }
+          "type": "nested",
```

Comment:
Doctors needs to be nested as, with the existing string array of doctors names, a search for 'Andrew Smith' with find a match in [ 'Dr Jane Smith', 'Dr Andrew Jones'].

The problem is described [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html).